web/src/pages/onboarding/OnboardingContainer.tsx

```diff
@@ -76,6 +76,9 @@ export const OnboardingContainer: React.FC<Props> = ({
     removeOnboardingProgress(dispatch, onboardingStep, currentCharacter)
   }
 
+  const jumpPrevOnboardingPage = () => {
+    removeOnboardingProgress(dispatch, onboardingStep, currentCharacter, 2)
+  }
```

Comment:
Please add inline comments with the rationale for this, will not be self-evident to future generations.

Reply:
Added.